src/docs/asciidoc/15-examples.adoc

```diff
@@ -30,6 +30,52 @@ task buildImage(type: DockerBuildImage) {
 }
 ----
 
+==== Interacting With a Container Running in the Background
+
+This code demonstrates managing a container running in the background with this plugin
+which can facilitate locally running a Dockerized application. This can be done by
+creating the container with an explicit `containerName` which is then used as
+the `targetContainerId` for relevant tasks.
+
+.build.gradle
+[source,groovy]
+----
+apply plugin: 'com.bmuschko.docker-remote-api'
+
+import com.bmuschko.gradle.docker.tasks.container.*
+
+ext {
+  CONTAINER_NAME = 'myapp'
+} 
+
+task buildMyAppImage(type: DockerBuildImage) {
+    inputDir = file('docker/myapp')
+    tag = 'test/myapp:latest'
+}
+
+task dockerCreateContainer(type: DockerCreateContainer) {
+    dependsOn buildMyAppImage
+    targetImageId { buildMyAppImage.imageId }
+    containerName = CONTAINER_NAME
+}
+
+task dockerStartContainer(type: DockerStartContainer) {
+    dependsOn dockerCreateContainer
+    targetContainerId { dockerCreateContainer.getContainerName() }
+}
+
+task dockerStopContainer(type: DockerStopContainer) {
+    targetContainerId { dockerCreateContainer.getContainerName() }
+}
+
+task dockerRemoveContainer(type: DockerRemoveContainer) {
+    targetContainerId { dockerCreateContainer.getContainerName() }
+}
+
+// Remove the container to allow reuse of the name.
+dockerStopContainer.finalizedBy dockerRemoveContainer
```

Comment:
Can you explain the use case you are modeling here? Should the build _always_ stop and remove the container? Or would this be a use case where you start a container by running a distinct build and then may sometime later stop and remove the container but manually interacting with the container? It's not super clear from the description. We should make it more clear.

Reply:
I'll look at tightening up the description of the use case. This bit is a workaround for `autoRemove` (https://github.com/bmuschko/gradle-docker-plugin/issues/599) which is all part of a way to manage ephemeral development containers.

Reply:
Thanks, I think letting users understand what problem were are trying to solve with the code example is very important. If this is more of a workaround then we should fix the underlying issue instead (if possible).

Reply:
if I have time this week I'll look at grabbing that issue.

Reply:
@mwhipple Can we close this PR given that it is a workaround for not having `autoRemove`?